semeio/jobs/correlated_observations_scaling/job_config.py

```diff
@@ -93,67 +169,36 @@ def build_schema():
             "CALCULATE_KEYS": {
                 MK.Required: True,
                 MK.Type: types.NamedDict,
+                MK.Description: _CALCULATE_KEYS_DESCRIPTION,
                 MK.Content: {
-                    "keys": {
-                        MK.Required: True,
-                        MK.Type: types.List,
-                        MK.Content: {
-                            MK.Item: {
-                                MK.Type: types.NamedDict,
-                                MK.Content: {
-                                    "key": {MK.Required: True, MK.Type: types.String},
-                                    "index": {
-                                        MK.Required: False,
-                                        MK.Type: types.List,
-                                        MK.LayerTransformation: _to_int_list,
-                                        MK.Content: {
-                                            MK.Item: {
-                                                MK.Type: types.Integer,
-                                                MK.ElementValidators: (_min_value,),
-                                            }
-                                        },
-                                    },
-                                },
-                            }
-                        },
-                    },
+                    "keys": _KEYS_SCHEMA,
                     "threshold": {
                         MK.Required: False,
                         MK.Type: types.Number,
                         MK.ElementValidators: (_min_max_value,),
+                        MK.Description: "Instructs the job to ignore principal"
+                        " components where the summative variance is smaller"
```

Comment:
Perhaps use a different word than summative? Not very commonly used.

Reply:
changed with `cummulative`